src/components/InlineEdit/CancelButton.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render } from 'enzyme';
+import { CancelButton } from './index';
+
+test('it renders properly', () => {
+  const component = render(<CancelButton />);
```

Comment:
can we use `shallow` instead for all snapshots now (i.e. tests that are not interactive should not require `render`)? See WizardPattern...